From ed6bc77e73ba858dd2ae17aa9dfc4eaee76caf83 Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Tue, 24 Mar 2020 14:12:00 +0530
Subject: [PATCH] Minor theme improvements

Color StatusBar and NavigationBars in most of the cases (both Day and night)
Disable AllCaps in some places
Tweak some colors
---
 app/src/main/res/values-night/themes.xml | 10 ++++++++--
 app/src/main/res/values/colors.xml       |  6 +++---
 app/src/main/res/values/themes.xml       | 15 ++++++++++++---
 3 files changed, 23 insertions(+), 8 deletions(-)

diff --git a/app/src/main/res/values-night/themes.xml b/app/src/main/res/values-night/themes.xml
index fbd6b7a04..62417160e 100644
--- a/app/src/main/res/values-night/themes.xml
+++ b/app/src/main/res/values-night/themes.xml
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <style name="Theme.SeriesGuide.DayNight" parent="Theme.SeriesGuide.Dark" />
-    <style name="Theme.SeriesGuide.DayNight.Immersive" parent="Theme.SeriesGuide.Dark.Immersive" />
+    <style name="Theme.SeriesGuide.DayNight" parent="Theme.SeriesGuide.Dark" >
+        <item name="android:navigationBarColor">@color/sg_color_background_dark</item>
+        <item name="android:statusBarColor">@color/sg_background_app_bar_dark</item>
+    </style>
+    <style name="Theme.SeriesGuide.DayNight.Immersive" parent="Theme.SeriesGuide.Dark.Immersive" >
+        <item name="android:navigationBarColor">@color/sg_color_background_dark</item>
+        <item name="android:statusBarColor">@color/sg_background_app_bar_dark</item>
+    </style>
 
 </resources>
\ No newline at end of file
diff --git a/app/src/main/res/values/colors.xml b/app/src/main/res/values/colors.xml
index 4ffd6cc91..43f81f683 100644
--- a/app/src/main/res/values/colors.xml
+++ b/app/src/main/res/values/colors.xml
@@ -33,7 +33,7 @@
     <color name="sg_color_secondary">@color/sg_color_secondary_light</color>
 
     <!-- Dark background is 8% primary on top of #121212 -->
-    <color name="sg_color_background_dark">#19161e</color>
+    <color name="sg_color_background_dark">#121212</color>
     <color name="sg_color_background_light">#ebebeb</color>
     <color name="sg_color_background">@color/sg_color_background_light</color>
 
@@ -50,8 +50,8 @@
 
     <color name="grey_50">#fffafafa</color>
     <color name="grey_850">#ff303030</color>
-    <color name="widget_default_background">@color/grey_850</color>
-    <color name="widget_default_background_light">@color/grey_50</color>
+    <color name="widget_default_background">#ff202026</color>
+    <color name="widget_default_background_light">#80ffffff</color>
 
     <color name="background_black">#ff000000</color>
     <color name="background_snackbar">#ff323232</color>
diff --git a/app/src/main/res/values/themes.xml b/app/src/main/res/values/themes.xml
index 025e85115..91fb67ba9 100644
--- a/app/src/main/res/values/themes.xml
+++ b/app/src/main/res/values/themes.xml
@@ -10,6 +10,8 @@
     <style name="Theme.BaseSeriesGuide.Dark" parent="Theme.Material3.Dark.NoActionBar">
 
         <!-- All customizations that are SPECIFIC to API-level 21 go here. -->
+        <item name="android:navigationBarColor">@color/sg_color_primary_dark</item>
+        <item name="android:statusBarColor">@color/sg_color_primary_900</item>
 
     </style>
 
@@ -52,6 +54,9 @@
         <item name="android:windowContentTransitions">true</item>
         <item name="windowActionModeOverlay">true</item>
 
+        <item name="android:navigationBarColor">@color/sg_color_background_dark</item>
+        <item name="android:statusBarColor">@color/sg_color_primary_900</item>
+
         <!-- Remove default window shadow -->
         <item name="android:windowContentOverlay">@null</item>
 
@@ -62,7 +67,7 @@
         <item name="sgColorIconInactive">@color/sg_white_50</item>
         <item name="sgColorDivider">@color/sg_white_15</item>
         <item name="sgColorShadow">@color/sg_black_40</item>
-        <item name="sgColorStatusBarOverlay">@color/sg_black_87</item>
+        <item name="sgColorStatusBarOverlay">@color/sg_color_primary_900</item>
         <item name="sgColorTopAppBar">@color/sg_background_app_bar_dark</item>
 
         <!-- dialogTheme is used by AppCompatDialogFragment. -->
@@ -80,7 +85,8 @@
         <!-- All customizations that are SPECIFIC to API-level 21 go here. -->
 
         <!-- Note: nav bar color needs to be opaque! -->
-        <item name="android:navigationBarColor">@color/sg_black_almost</item>
+        <item name="android:navigationBarColor">@color/sg_color_primary_dark</item>
+        <item name="android:statusBarColor">@color/sg_color_primary_900</item>
 
     </style>
 
@@ -119,6 +125,9 @@
         <item name="colorSurfaceInverse">@color/md_theme_light_inverseSurface</item>
         <item name="colorPrimaryInverse">@color/md_theme_light_primaryInverse</item>
 
+        <item name="android:navigationBarColor">@color/sg_white</item>
+        <item name="android:statusBarColor">@color/sg_color_primary_900</item>
+
         <!-- Windows -->
         <item name="android:windowBackground">@color/sg_color_background</item>
         <item name="android:windowContentTransitions">true</item>
@@ -134,7 +143,7 @@
         <item name="sgColorIconInactive">@color/sg_black_38</item>
         <item name="sgColorDivider">@color/sg_black_10</item>
         <item name="sgColorShadow">@color/sg_black_7</item>
-        <item name="sgColorStatusBarOverlay">@color/sg_white_87</item>
+        <item name="sgColorStatusBarOverlay">@color/sg_color_primary_900</item>
         <!-- This must not match colorSurface or AppBarLayout uses color to elevate itself. -->
         <item name="sgColorTopAppBar">@color/sg_color_background</item>
 
